src/main/java/de/tum/in/www1/artemis/web/rest/lecture/LectureUnitResource.java

```diff
@@ -120,8 +122,8 @@ public LectureUnitResource(AuthorizationCheckService authorizationCheckService,
         if (!lectureUnit.getLecture().getId().equals(lectureId)) {
             return conflict();
         }
-        if (!authorizationCheckService.isAtLeastEditorInCourse(lectureUnit.getLecture().getCourse(), null)) {
-            return forbidden();
+        if (!authorizationCheckService.isAtLeastInstructorInCourse(lectureUnit.getLecture().getCourse(), null)) {
```

Comment:
please use the variant `...elseThrow`. Then you don't need the if statement

Reply:
Thanks for the heads up, this is addressed in https://github.com/ls1intum/Artemis/pull/4201/commits/6f97a6ac86dd7cf84326cfaa594e8478061a2b4a 
I was not aware of the `checkHasAtLeastRoleInCourseElseThrow` method in the `AuthorizationCheckService`